ffi/capi/examples/fixeddecimal/Makefile

```diff
@@ -16,10 +16,50 @@ $(ALL_HEADERS):
 ../../../../target/debug/libicu_capi.a: $(ALL_RUST)
 	cargo build
 
+../../../../target/x86_64-unknown-linux-gnu/debug/libicu_capi.a: $(ALL_RUST)
+	RUSTFLAGS="-Clinker-plugin-lto -Clinker=clang -Clink-arg=-flto -Cpanic=abort" cargo +nightly-2021-02-28 panic-abort-build --target x86_64-unknown-linux-gnu --no-default-features --features x86tiny
+
+../../../../target/x86_64-unknown-linux-gnu/release/libicu_capi.a: $(ALL_RUST)
+	RUSTFLAGS="-Clinker-plugin-lto -Clinker=clang -Clink-arg=-flto -Cpanic=abort" cargo +nightly-2021-02-28 panic-abort-build --target x86_64-unknown-linux-gnu --no-default-features --features x86tiny --features smaller_static --release
+
+# Naive target: no optimizations, full std
 a.out: ../../../../target/debug/libicu_capi.a $(ALL_HEADERS) test.c
 	gcc test.c ../../../../target/debug/libicu_capi.a -ldl -lpthread -lm -g
 
-build: a.out
+# optim.elf: gcc with maximum link-time code stripping (gc-sections and strip-all)
```

Comment:
nit: could this stuff all be made into a separate `make test-tiny` target?, and moved further down the file? That way we can merge this and it does not impact regular testing.